src/compiler/scala/tools/nsc/SubComponent.scala

```diff
@@ -50,7 +50,7 @@ abstract class SubComponent {
   /** True if this phase runs after all other phases. Usually, `terminal`. */
   val terminal: Boolean = false
 
-  /** SubComponent are added to a HashSet and two phases are the same if they have the same name  */
+  /** SubComponent are added to a HashSet and two phases are the same if they have the same name. */
```

Comment:
SubComponent[s]. Definitely appreciate the period. `scaladoc --periodicize` could assume it.

Reply:
```suggestion
  /** SubComponents are added to a HashSet and two phases are the same if they have the same name. */
```